pyaedt/modules/SetupTemplates.py

```diff
@@ -1511,23 +1511,21 @@ def add_subrange(self, rangetype, start, end=None, count=None, unit="GHz", save_
             self.props["SweepRanges"] = {"Subrange": []}
             return self.update()
 
-        range = {}
-        range["RangeType"] = rangetype
-        range["RangeStart"] = str(start) + unit
+        interval = {"RangeType": rangetype, "RangeStart": str(start) + unit}
```

Comment:
@MaxJPRey I actually couldn't rename as subrange because it was a type name too, so I went for interval. Hopefully is clear enough :)

Reply:
@gmalinve yes, this is very good.